conformance/tests/conformance_spec.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: conformance
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: conformance
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - pods
+  - secrets
+  - services
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - update
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+- apiGroups:
+  - gateway.networking.k8s.io
+  resources:
+  - gatewayclasses
+  verbs:
+  - get
+  - list
+- apiGroups:
+  - gateway.networking.k8s.io
+  resources:
+  - gateways
+  - httproutes
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: nginx-conformance
+subjects:
+- kind: ServiceAccount
+  name: conformance
+  namespace: default
+roleRef:
+  kind: ClusterRole
+  name: conformance
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: conformance
+spec:
+  containers:
+  - name: conformance
+    image: conformance-test-runner:latest
+    imagePullPolicy: Never
+    env:
+      - name: GATEWAY_CLASS
```

Comment:
nit: inconsistent indentation for `-`:

```
   containers:
  - name: conformance
```
and
```
    env:
      - name: GATEWAY_CLASS
```

Reply:
Changed because of separate changes